dev-env/bin/daml-sdk-head

```diff
@@ -105,9 +105,9 @@ function cleanup() {
 trap cleanup EXIT
 
 # Building here separately so the user can see the build process which could take a while
-bazel build "${BAZEL_MODE_FLAGS[@]}" //release:sdk-release-tarball.tar.gz
+bazel build ${BAZEL_MODE_FLAGS[@]:-} //release:sdk-release-tarball.tar.gz
 
-readonly TARBALL="$(bazel info bazel-bin "${BAZEL_MODE_FLAGS[@]}")/release/sdk-release-tarball.tar.gz"
+readonly TARBALL="$(bazel info bazel-bin "${BAZEL_MODE_FLAGS[@]:-}")/release/sdk-release-tarball.tar.gz"
```

Comment:
Do you wanna remove the quotes here too?

Reply:
Yes. I had missed the closing paren so thought the variable interpolation was part of the path. Thanks for pointing this out.

Reply:
```
$ bazel info bazel-bin ""
INFO: Invocation ID: 948b0978-7edc-4fe9-aed5-97e2c698b73e
ERROR: at most one key may be specified
$
```
Fails indeed.